admin/resources/js/components/pool/CreatePool.tsx

```diff
@@ -51,7 +51,7 @@ export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
   handleCreatePool,
 }) => {
   const intl = useIntl();
-  const locale = getLocale(intl.locale);
+  const locale = getLocale(intl);
```

Comment:
The `getLocale(locale: string)` method requires the locale not the intl hook, so it should be: 

`const locale = getLocale(intl.locale)`

Reply:
Nvm, the localize function has been changed to accept the entire IntlShape. So, this is fine 👍.